docs/gaia/join-testnet.md

```diff
@@ -103,6 +103,37 @@ curl https://raw.githubusercontent.com/cosmos/testnets/master/latest/genesis.jso
 Note we use the `latest` directory in the [testnets repo](https://github.com/cosmos/testnets)
 which contains details for the latest testnet. If you are connecting to a different testnet, ensure you get the right files.
 
+You now need to generate a `genesis tx` to generate a signed self-delegation:
+
+```bash
+gaiad gentx \
+    --amount <amount_of_delegation> \
+    --commission-rate <commission_rate> \
+    --commission-max-rate <commission_max_rate> \
+    --commission-max-change-rate <commission_max_change_rate> \
+    --pubkey <consensus_pubkey> \
+    --name <key_name>
+```
+
+If unspecified, `consensus_pubkey` will default to the output of `gaiad tendermint show-validator`.
+`key_name` is the name of the private key that will be used to sign the transaction.
```

Comment:
There is no `key_name` in the above command. Do you mean `name`? Should this be changed to be more clear?

Reply:
I meant `key_name`:

```
gaiad gentx \
    --amount <amount_of_delegation> \
    --commission-rate <commission_rate> \
    --commission-max-rate <commission_max_rate> \
    --commission-max-change-rate <commission_max_change_rate> \
    --pubkey <consensus_pubkey> \
    --name <key_name>
```